src/Actions/Family/ListFamilies.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Choredo\Actions\Family;
+
+use Assert\Assert;
+use Choredo\Entities\Family;
+use Choredo\EntityManagerAwareInterface;
+use Choredo\Filter;
+use Choredo\Filterable;
+use Choredo\HasEntityManager;
+use Choredo\Output\CreatesFractalScope;
+use Choredo\Output\FractalAwareInterface;
+use Choredo\Pageable;
```

Comment:
I like to put these into a namespace together, usually something like `Choredo\Actions\Behaviors` or `Choredo\Actions\Contracts` Thoughts?